.github/abstract/How to use SwiftCurrent with SwiftUI.md

```diff
@@ -0,0 +1 @@
+SwiftCurrent has first-class SwiftUI support. It’s designed to naturally feel like using any other SwiftUI tool. It is optimized to avoid common pitfalls. For example, SwiftCurrent does not use `AnyView` in its implementation. This means all of the animation and performance benefits you get when using SwiftUI normally you get when using SwiftCurrent.
```

Comment:
The last sentence caused me to stumble a little bit.  I think this is better for now.
```suggestion
SwiftCurrent has first-class SwiftUI support. It’s designed to naturally feel like using any other SwiftUI tool. It is optimized to avoid common pitfalls. For example, SwiftCurrent does not use `AnyView` in its implementation. This means that, when using SwiftCurrent, you will keep all the animation and performance benefits you would normally have gotten.
```